tutorial/about-tutorial.md

```diff
@@ -1,101 +1,58 @@
-* A conceptual understanding about RESTful APIs. For more information, see http://example.com/restful_apis.
-* (Optional) A development environment (IDE) that displays API responses formatted for readability.
-* A list of favorites prepared, so you can manage them. For more information about favorites lists, see http://example.com/favorite_lists.
+This template breaks down the tutorial into _parts_. One _part_ of the tutorial may focus on completing several related steps. You're welcome to follow this structure or use the step headings on their own.
 
-```
+#### Tips for writing steps in a tutorial
 
-### The Part sections
+* When writing your steps, make sure to provide any background information your users may need to complete the task. 
+* Remember to orient your users when walking them through each step. If they need to open a particular file or dialog to complete the task, provide that information first.
 
-Use either a structured style, or an informal style in this section.
+### About the "Summary" section
 
-#### Structured style
+Use the summary section of the tutorial to sum up what tasks your user completed during the tutorial and what your users now know how to do.
 
-The structured approach features distinct headings for each step part:
+### About the "Next Steps" section
 
-```markdown
-Part One
-  Goal
-  Procedure
-  Outcome
-Part Two
-  Goal
-  Procedure
-  Outcome
-```
+Use this section to include links to other tutorials, such as tutorials that allow users to learn about other features. You can also include links to relevant resources, like articles, blogs, or videos.
 
-This structure works well for beginners because it describes what each part teaches them, and what they should expect at the end of each part.
+## Helpful tips when writing your tutorial 
 
-This structure also works well for tutorials separated into parts, or for complex tutorials that contain a lot of steps.
-
-#### Informal style
-
-The informal style is conversational, and does not include goal or outcome statements:
-
-```markdown
-Narrative flow
-Conversational procedural structure, with ordered steps described in the body.
-No outcome statement.
-```
-
-This structure works well for less complex tutorial topics.
-Use this style for simple lessons that contain fewer steps and a basic concept.
-You get a more conversational flow with this tutorial style, although it loses some of the prescriptive clarity by not including headings.
-
-## Tutorial examples
-
-* **Example 1**.
-
-* **Example 2**.
+* Keep your tutorial focused. If needed, include links to other pieces of documentation that explain concepts or terms in more depth.
+* If users will copy and paste code samples, make sure they include the following:
+  * Any required `import` or `using` statements
+  * Code comments that explain what the code does 
+* Lengthy tutorials can overwhelm users. Consider condensing or removing steps or reevaluating the scope of the tutorial.
```

Comment:
**Suggestion (non-blocking):** Maybe general tutorial writing tips should go earlier in the document before you get into the tips for each section? Also, I think it would be helpful to provide an example of a code snippet that includes a code comment.